astro/src/content/docs/apis/_user-login-response.mdx

```diff
@@ -1,29 +1,47 @@
 import APIField from 'src/components/api/APIField.astro';
 import InlineField from 'src/components/InlineField.astro';
 
+<APIField name="emailVerificationId" type="String">
```

Comment:
leaving out the nuance on both of these, that the tenant has to be set to formfield when the login occurs AND the identifier needs to have an OTP on it (identifier could have an OTP on it but strategy has since been changed.)

Reply:
That's right. Not sure why it was missed before.